Woof/Woof/Model/Sitter.swift

```diff
@@ -0,0 +1,32 @@
+import Foundation
+
+/**
+ A person who can walk with pet (or several pets) and provide service of walking for pet owners.
+ Conforms to the `Codable` and `Identifiable` protocols.
+ */
+struct Sitter: Codable, Identifiable {
+    /// The unique identifier for the sitter.
+    let id: UUID
+
+    /// The name of the sitter.
+    var name: String
+
+    /// The surname of the sitter.
+    var surname: String
+
+    /// The phone number of the sitter.
+    var phone: String
+
+    /// The URL of the sitter's avatar image.
+    var avatarUrl: URL?
+
+    /// The additional information about the sitter, like his experience with dogs, favorite places for walks,
```

Comment:
nit: girls can be sitter too :)
Please use gender-neutral words where it is possible.

```suggestion
    /// The additional information about the sitter, like their experience with dogs, favorite places for walks,
```